### DISCUSSION TITLE
Add GoAccess statistics option to NPM LXC script

### DISCUSSION THREAD

**Discussion by Saik0Shinigami** [tteck/Proxmox#2245] [Ideas]
A GoAccess addon option for Nginxproxymanager much like portainer is an option for the docker container

https://github.com/xavier-hernandez/goaccess-for-nginxproxymanager

**Comment by gerard33** [comment]
Adding GoAccess as an excellent idea.

I have installed GoAccess on the NPM LXC following these steps --> https://goaccess.io/download#official-repo
and then configured GoAccess using these instructions --> https://www.reddit.com/r/selfhosted/comments/swi4lf/comment/kb8efua/

**Reply by Saik0Shinigami** [reply]
Yea those instructions work EXCEPT that you need to make the /app/goaccess folder before you run it daemonized, otherwise it complains that the folder doesn't exist and can't generate the index.html file.
